Basic/Pod/DeveloperGuide.pod

```diff
@@ -57,117 +57,79 @@ This is detailed in section 4.11 of L<PDL::FAQ>.
 
 =head1 PDL Developer Notes
 
-=head2 Random Musings
+=head2 Getting (some) Debug Information
 
-A (small) collection of random musings to note if you feel
-the need to improve or add to PDL (please do):
+PDL has two global variables which control debugging output while
+running:
 
 =over
 
-=item Organizing Commits
+=item C<$::PP_VERBOSE>
 
-git supports file-by-file commits so it is helpful
-to commit your changes to git a little at a time where
-each commit corresponds to a single change.  This makes
-it easy in the log to determine what was done and to
-locate any desired commit in case of issues that need
-to be resolved.
+If set to a true value, L<PDL::PP> will print details while processing
+C<pp_def>.
```

Comment:
This is not really while "running" but while building. Those are both things that a developer is going to care about, but I'd like to be as accurate as possible while we're updating these docs :-)

Reply:
There's also a `$PDL::verbose` which is worth mentioning.